packages/features/form-builder/Components.tsx

```diff
@@ -512,5 +512,14 @@ export const Components: Record<FieldType, Component> = {
       );
     },
   },
+  url: {
+    propsType: propsTypes.url,
+    factory: (props) => {
+      if (!props) {
```

Comment:
Is there a case where props could be `undefined`? It feels wrong to return a different element from the same component conditionally. Like if no props are provided return <div> otherwise <TextWidget>